### ISSUE TITLE
Bug in style transform

### ISSUE THREAD

**Issue by macdonst** [enhance-dev/enhance-custom-element-mixin#4]
I found a bug in the style transform. If you do:

```css
          todo-app {
```

it becomes:

```css
          todo-app todo-app {
```

When it should skip adding the custom element name if it already exists.

**Comment by ryanbethel** [comment]
I don't think this is a bug. The SSR behavior is the same. If you want to style the element itself you can use the :host selector. Every rule gets the element name prepended unless it starts with :host.

**Comment by macdonst** [comment]
@ryanbethel is right, this is the expected behaviour for style transform but the question now is, should we fix it?

**Comment by kristoferjoseph** [comment]
I ran into this when I was working on the other versions of the shady styles transform.
My opinion is we should fix this since it is a way to shortcut the need of a `:host` transform.
We could inform users to use `scope=global` when wanting to use the hardcoded custom element tag selector in their styles.
Currently using the custom element tag name instead of `:host` in your styles is an anti-pattern as you could specify any custom element tag name when you register the class in `customElements.define` or the Enhance element call of `enhance('my-element', {...})`
